### ISSUE TITLE
Action recognition classifier overfitting

### ISSUE THREAD

**Issue by braunjon** [anindita127/IMoS#10]
Hi,

I have trouble training the LSTM_Action_Classifier. It is quickly overfitting to the train-split while the loss for the val-split is going up.
I am not sure if this is due to splitting the data based on subjects. Did you use the same train (s2-s8), val (s1), test (s10) splits to train the action classifier?

Thanks for your help

**Comment by btekin** [comment]
Seeing the exact same issue. 

Could you provide any pointers (train/test split, training scheme, any input processing) to be able to reproduce the action recognition training in the paper?

**Comment by anindita127** [comment]
Hi,

Thanks for your interest in our work. The idea of the action classifier was to train it on the groundtruth actions and test them on the synthesised actions. So we used the whole dataset to train the classifier. Hope this helps!
